browse/static/css/arXiv.css

```
@@ -827,6 +827,31 @@ p.tagline {
   cursor: pointer;
 }
 
```

add a note for future editors:
```suggestion
/*added by Papers With Code*/
```